test/main-test.js

```diff
@@ -141,6 +153,27 @@ describe('PercyClient', function() {
           done(err);
         });
     });
+
+    it('returns build data without being given a project identifier', function(done) {
+      nock('https://percy.io')
+        .post('/api/v1/builds/')
+        .reply(201, responseMock);
+
+      // intermediate state while we move to token-only build creation
+      let request = percyClient.createBuild('', {
```

Comment:
Would be slightly more idiomatic to pass `null` or even `undefined` here.

Reply:
The other very-JavaScripty way to do this is to modify the createBuild method to check the length of `arguments` and if it's 2, assign both project and options, if it's only 1 (or if the first object `typeof == object` instead of a string), assign it to options. That way the method can accept either `createBuild({options: ...})` or the legacy way.

Reply:
I like that